www/js/lib/zimfile.js

```diff
@@ -54,14 +54,6 @@ if (!String.prototype.startsWith) {
     });
 }
 
-/**
- * A global variable to track the assembler machine type and the last used decompressor (for reporting to the API panel)
- * This is populated in the Emscripten wrappers
- * @type {Object}
- * @property {String} assemblerMachineType The assembler machine type supported and/or loaded by this app: 'ASM' or 'WASM'
- * @property {String} decompressorLastUsed The decompressor that was last used to decode a compressed cluster (currently 'XZ' or 'ZSTD')
- * @property {String} errorStatus A description of any detected error in loading a decompressor
- */
```

Comment:
I see you've moved the typedef to init.js, which probably makes sense (though you've given it the wrong description there). However, please leave the explanation of this object property (lines 57-59), and refer the dev to the object definition in init.js. Also change "A global variable" to "A global parameter object to track".